### ISSUE TITLE
Feature Request: Bulk rename/replace colour variables and styles 

### ISSUE THREAD

**Issue by rossul** [Ashung/Automate-Sketch#304]
Thanks for the great plugins. 
Being able to bulk rename, find&replace names for colour variables and styles would be awesome.

**Comment by everydayslang** [comment]
Yeah, just realized this wasn't in there, which is weird, because Automate basically does everything I could ever think of.
